packages/fixie/src/isomorphic-client.ts

```diff
@@ -52,11 +52,13 @@ export class IsomorphicFixieClient {
 
   /**
    * Create a new FixieClient without an API key. This is only useful for accessing public APIs, such as the conversation APIs.
+   *
+   * You only need to pass url if you're pointing to a different Fixie backend than the default production. Unless you specificially know you need to do this, you don't.
    */
   // This is also useful for running in the console.fixie.ai webapp, because it's on the same host
   // as the backend and thus doesn't need the API key, assuming we set the auth cookies to be cross-domain.
-  static CreateWithoutApiKey(url: string) {
-    return new this(url);
+  static CreateWithoutApiKey(url?: string) {
```

Comment:
Why not just "Create" and have the API key parameter be optional?

Reply:
That would be fine too. When I made this, we had this above:

```ts
  static Create(url: string, apiKey?: string) {
    const apiKeyToUse = apiKey ?? process.env.FIXIE_API_KEY;
    if (!apiKeyToUse) {
      throw new Error(
        'You must pass apiKey to the constructor, or set the FIXIE_API_KEY environment variable. The API key can be found at: https://console.fixie.ai/profile'
      );
    }
    return new this(url, apiKey);
  }
```

Clearly, someone felt strongly that an API key was a good thing. :smile: 

I think creating a client without an API key is a rare enough case that it's worth asking the dev to be more intentional when they do it.